ui/components/ui/contract-token-values/contract-token-values.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CopyContractDetails from '../icon/copy-contract-details';
+import IconBlockExplorer from '../icon/icon-block-explorer';
+import Box from '../box/box';
+import Tooltip from '../tooltip/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Identicon from '../identicon/identicon.component';
+import Typography from '../typography/typography';
+import {
+  FONT_WEIGHT,
+  TYPOGRAPHY,
+  DISPLAY,
+  COLORS,
+  ALIGN_ITEMS,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+import Button from '../button';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+
+export default function ContractTokenValues({ address, tokenName }) {
+  const t = useI18nContext();
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  return (
+    <Box
+      display={DISPLAY.FLEX}
+      alignItems={ALIGN_ITEMS.CENTER}
+      justifyContent={JUSTIFY_CONTENT.CENTER}
+      className="contract-token-values"
+    >
+      <Box marginRight={2} marginTop={2} marginBottom={2}>
```

Comment:
Should need vertical margins here if `alignItems` is set to center?
```suggestion
      <Box marginRight={2}>
```

If it's to position it in layout the margins should be added when it's used either by a className or by spreading props so Box props can be used.

```
// Add margins when the component is being used in a layout
<ContractTokenValues address={...} tokenName={...} marginTop={2} marginBottom={3} />
```

Reply:
@georgewrmarshall Deleted vertical margins [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/48c24613122a69ea20b0ad6504a3586585b6be25).